lib/new_relic/agent/transaction/request_attributes.rb

```diff
@@ -24,7 +24,7 @@ def initialize(request)
           @referer = referer_from_request(request)
           @accept = attribute_from_env(request, HTTP_ACCEPT_HEADER_KEY)
           @content_length = content_length_from_request(request)
-          @content_type = attribute_from_request(request, :content_type)
+          @content_type = content_type_attribute_from_request(request)
```

Comment:
Optionally, you could simply leverage the existing `attribute_from_method`'s functionality instead of creating a special one-off method for content-type.

```suggestion
          @content_type = content_type_attribute_from_request(request)
          @content_type = attribute_from_request(request, :media_type) || attribute_from_request(request, :content_type)
```

Reply:
While I love the simplicity of just using one method, I liked having that longer comment + link explaining the change and it looked a bit off here.